.github/workflows/nodejs.yml

```diff
@@ -1,57 +0,0 @@
-name: Node CI
```

Comment:
I don't think the CI setup needs to remove. We should provide a case on how to set up a pipeline example, right?

Reply:
How is a pipeline really relevant to Preact? I would say no, we shouldn't.